client/css/style.css

```diff
@@ -978,6 +978,17 @@ background on hover (unless active) */
 	font-size: 14px;
 }
 
+#windows .header .topic-edit {
+	color: #222;
```

Comment:
Doing the following styles makes it work nicely on other themes (e.g. with dark background):

```css
    color: inherit;
    background: inherit;
```